### PR TITLE
login to main docker hub as well

### PR DESCRIPTION
**Pull request by hillalex**
Log the agents in to docker hub using the `vimcrobot` docker user

### CODE REVIEW

File: shared/files/agent/montagu-docker-login [hunk 1]
```diff
@@ -11,6 +11,7 @@ vault auth -method=github > /dev/null
 
 REGISTRY_URL=docker.montagu.dide.ic.ac.uk:5000
 REGISTRY_USER=vimc
+HUB_USER=vimc-robot
```
Comment (richfitz) [inline shared/files/agent/montagu-docker-login]: This should be vimcrobot I believe (were names including hyphens disallowed on docker hub?)

**Review by richfitz — CHANGES_REQUESTED** [review]

File: shared/files/agent/montagu-docker-login (hunk 1 above)
Comment (hillalex) [reply shared/files/agent/montagu-docker-login]: oops, yes. yes they were

File: shared/files/agent/montagu-docker-login [hunk 2]
```diff
@@ -20,3 +21,6 @@ fi
 
 echo $REGISTRY_PASSWORD | \
     docker login -u $REGISTRY_USER --password-stdin $REGISTRY_URL
+
+echo $REGISTRY_PASSWORD | \
+    docker login -u $HUB_USER --password-stdin
```
Comment (MartinEden) [inline shared/files/agent/montagu-docker-login]: We use the same password for both registries? Seems a bit insecure?

**Review by MartinEden — CHANGES_REQUESTED** [review]

File: shared/files/agent/montagu-docker-login [hunk 3]
```diff
@@ -11,16 +11,22 @@ vault auth -method=github > /dev/null
 
 REGISTRY_URL=docker.montagu.dide.ic.ac.uk:5000
 REGISTRY_USER=vimc
-HUB_USER=vimcrobot
 REGISTRY_PASSWORD=$(vault read -field=password /secret/registry/${REGISTRY_USER})
 
+HUB_USER=vimcrobot
+HUB_PASSWORD=$(vault read -field=password /secret/${HUB_USER$)
```
Comment (MartinEden) [inline shared/files/agent/montagu-docker-login]: Syntax error here I think. Also, we seem to have both a folder and a secret at `secret/vimcrobot` in the vault?

**Review by MartinEden — CHANGES_REQUESTED** [review]

File: shared/files/agent/montagu-docker-login (hunk 3 above)
Comment (hillalex) [reply shared/files/agent/montagu-docker-login]: Thanks, have deleted the folder

**Review by MartinEden — APPROVED** [review]

File: shared/files/agent/montagu-docker-login [hunk 4]
```diff
@@ -13,10 +13,20 @@ REGISTRY_URL=docker.montagu.dide.ic.ac.uk:5000
 REGISTRY_USER=vimc
 REGISTRY_PASSWORD=$(vault read -field=password /secret/registry/${REGISTRY_USER})
 
+HUB_USER=vimcrobot
+HUB_PASSWORD=$(vault read -field=password /secret/${HUB_USER)
```
Comment (richfitz) [inline shared/files/agent/montagu-docker-login]: To keep the vault somewhat tidy - would you mind moving this to `/secret/vimc-robot/dockerhub` - perhaps also add a `username=vimcrobot` field to that entry?

**Review by richfitz — CHANGES_REQUESTED** [review]

File: shared/files/agent/montagu-docker-login (hunk 4 above)
Comment (hillalex) [reply shared/files/agent/montagu-docker-login]: :+1:

**Comment by hillalex** [comment]
Cool, now reading username and pwd from vault

**Review by richfitz — APPROVED** [review]

**Pull request merged**